static/.eslintrc.js

```diff
@@ -0,0 +1,255 @@
+/* @file eslintrc
+ * @brief ESLintRC for ScottyLabs JS code.
+ *
+ * This is a pretty freaking (or flippin', as our Tech Director would say)
+ * pedantic set of rules.  100% compliance is in no way required, but it helps
+ * find some egregious stuff (and also helps with some subtle bugs that might be
+ * otherwise hard to catch.
+ *
+ * @author Oscar Bezi (bezi@scottylabs)
+ */
+
+'use strict';
+
+module.exports = {
+  'ecmaFeatures': {
+    'arrowFunctions': true,
+    'jsx': true,
```

Comment:
I was curious and looked into this but I couldn't actually find any that use this. Maybe I didn't look hard enough, but do you have an example you're thinking of?